plugins/inputs/win_wmi/query.go

```diff
@@ -0,0 +1,158 @@
+//go:build windows
```

Comment:
Any logic changes or just copy and paste?

Reply:
Just copy-and-paste plus dissolving some functions and copy the code to the corresponding calling location.